src/Arith.jl

```diff
@@ -254,7 +283,7 @@ Base.:-(x::Cyclotomic{<:NfPoly}, y::Union{Int64, Rational{Int64}}) = x-(y*one(x)
 Base.:-(y::Cyclotomic) = (-1)*y
 
 Base.:(==)(x::CycloSum, y::CycloSum) = iszero(x-y)
-Base.:(==)(x::CycloFrac, y::CycloFrac) = iszero(x-y)
+Base.:(==)(x::CycloFrac, y::CycloFrac) = iszero(x-y)  # TODO deal with exceptions?
```

Comment:
If you change `iszero` to honor exceptions, this one is fine as well.
```suggestion
Base.:(==)(x::CycloFrac, y::CycloFrac) = iszero(x-y)
```

Reply:
If I implemented your suggestion regarding `iszero` any `CycloFrac` with at least one exception wouldn't even be equal to itself anymore as far as I understand it.

Reply:
Ah, true. 

For this and other applications it would be good to have a variant of is_zero which ignores the exceptions.